app/components/status_card/view.html.erb

```diff
@@ -0,0 +1,6 @@
+<a class="app-status-card app-status-card--<%= status_colour %> app-status-card__link" href=<%= target %>>
```

Comment:
You could also use the rails helper `link_to` here - it can accept a block of html. But it's no biggie.

Reply:
I've kept as is!